src/main/resources/templates/owners/findOwners.html

```diff
@@ -26,8 +26,30 @@ <h2>Find Owners</h2>
           Owner by Last Name</button>
       </div>
     </div>
+	</form>
+	
+	<form th:object="${owner}" th:action="@{/owners}" method="get"
```

Comment:
use th:block on form here here to hide when your toggle is enabled `toggles[3]`

Look at form below this one for reference